jest.setup.js

```diff
@@ -1,2 +1,3 @@
 /* global jest */
 jest.setTimeout(120000);
+jest.retryTimes(3)
```

Comment:
https://jestjs.io/docs/jest-object#jestretrytimesnumretries-options

bulk tests are idempotent so we can do this now. Will also help when a test fail due to an intermittent API error (like `SEARCH NOT AVAILABLE`).